src/main/scala/cromwell/engine/backend/jes/JesBackend.scala

```diff
@@ -119,12 +119,13 @@ class JesBackend extends Backend with LazyLogging with ProductionJesAuthenticati
 
   private def writeAuthenticationFile(workflow: WorkflowDescriptor) = authenticated { connection =>
     val path = GoogleCloudStoragePath(gcsAuthFilePath(workflow))
+    val log = workflowLogger(workflow)
 
     generateAuthJson(jesConf.dockerCredentials, getGcsAuthInformation(workflow)) foreach { content =>
       def upload(prev: Option[Unit]) = connection.storage.uploadJson(path, content)
 
-      logger.info(s"Creating authentication file for workflow ${workflow.id} at \n ${path.toString}")
-      withRetry(upload, s"${makeTag(workflow)} Exception occurred while uploading auth file to $path")
+      log.info(s"Creating authentication file for workflow ${workflow.id} at \n ${path.toString}")
+      withRetry(upload, s"${log.makeTag} Exception occurred while uploading auth file to $path")
```

Comment:
So, presumably this text is for a "retry still failed" log message? If so, is *that* log message going to the correct log files? The fact that you're including a tag here makes me suspicious that it'd be lost in the main log only.